### DISCUSSION TITLE
Anonymous downloads

### DISCUSSION THREAD

**Discussion by PH5HP** [rejetto/hfs#710] [General]
Hi,

Is it possible to restrict the amount of data an anonymous ip adress can download? If not: can you add such an option?

Kind regards,
Henk - https://fileserver.ph5hp.nl - https://ph5hp.nl

**Comment by rejetto** [comment]
HFS doesn't have anything like that, but it is possible that the
download-quota plugin can be expanded in this sense

**Reply by PH5HP** [reply]
Hi,

That would be nice...

Kind regards,
Henk - https://fileserver.ph5hp.nl/ - https://ph5hp.nl/

**Comment by rejetto** [comment]
ip can change easily, often just by restarting your modem.
i'm not sure it would be worth to offer that.

**Reply by PH5HP** [reply]
Hi,

Yes... You are right about that,
